module/behaviour/skills/FallingRelax/data/config/FallingRelax.yaml

```diff
@@ -1,6 +1,8 @@
-{
-	"FALLING_ANGLE" : 0.8,
-	"FALLING_ACCELERATION" : 8,
-	"RECOVERY_ACCELERATION" : [9.5, 10.5],
-	"PRIORITY" : 90
-}
+# Common
+log_level: WARN
+relax_fall_priority: 5
+falling_angle: 0.65 #0.611
```

Comment:
Stating that this value is in Radians in a comment (as well as falling_acceleration) would be useful for newer members trying to read through these config files.